src/inference/Inference.jl

```diff
@@ -369,12 +374,55 @@ function AbstractMCMC.bundle_samples(
     )
 end
 
+function AbstractMCMC.bundle_samples(
+    rng::AbstractRNG,
+    model::AbstractModel,
+    spl::Sampler,
+    N::Integer,
+    ts::Vector{<:AbstractTransition},
+    chain_type::Type{Any};
+    discard_adapt::Bool=true,
+    save_state=true,
+    kwargs...
+)
+    return ts
+end
```

Comment:
I thought the `Any` case should be covered by the [default implementation in AbstractMCMC](https://github.com/TuringLang/AbstractMCMC.jl/blob/0641b816c91027f0f1ffaf3c515ac103b5b45b68/src/AbstractMCMC.jl#L272-L282)?

Reply:
You are correct, good catch!